Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -1317,14 +1318,18 @@ def panorama_create_address_command(args: dict):
     fqdn = args.get('fqdn')
     ip_netmask = args.get('ip_netmask')
     ip_range = args.get('ip_range')
+    ip_wildcard = args.get('ip_wildcard')
 
-    if not fqdn and not ip_netmask and not ip_range:
-        raise Exception('Please specify exactly one of the following: fqdn, ip_netmask, ip_range.')
+    # make sure only one of fqdn/ip_netmask/ip_range/ip_wildcard was provided.
 
-    if (fqdn and ip_netmask) or (fqdn and ip_range) or (ip_netmask and ip_range):
-        raise Exception('Please specify exactly one of the following: fqdn, ip_netmask, ip_range.')
+    if sum(
+        map(bool, [(fqdn is not None), (ip_netmask is not None), (ip_range is not None), (ip_wildcard is not None)])
+    ) != 1:
+        raise DemistoException(
+            f'Please specify exactly one of the following arguments: fqdn, ip_netmask, ip_range, ip_wildcard.'
```

Comment:
you cant have a range with a subnet? eg 10.0.15.0 - 10.0.15.15  / 255.255.255.0

Reply:
<img width="903" alt="image" src="https://user-images.githubusercontent.com/53861351/191678985-13121228-7b25-43f8-8180-9387ffa21705.png">

you can't have a range with a subnet, only one of them is allowed